packages/ui/src/components/vuestic-components/va-select/VaSelect.vue

```diff
@@ -357,14 +359,14 @@ export default class VaSelect extends mixins(
   }
 
   selectOption (option: any): void {
-    const isSelected = this.checkIsOptionSelected(option)
-
     if (this.doShowSearchInput) {
       (this as any).$refs.searchBar.focus()
```

Comment:
Simply remove this line. As you have an issue with focusing the search bar after an option selected, you need just remove the search bar focusing (on this line).

```
(this as any).$refs.searchBar.focus()
```